push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/holder/MessageHolderWithVariants.java

```diff
@@ -34,6 +34,8 @@
 
     private static final long serialVersionUID = -7955411139315335655L;
 
+    public static final int INITIAL_SERIAL_ID = 0;
```

Comment:
I will add comment, that's the serialId for first token batch. We use it in tests, so I wanted to avoid that be a magic constant